src/dataset-builder/ConceptSearch.ts

```diff
@@ -98,30 +98,32 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
             rowStyle: {
               backgroundColor: 'white',
               ...tableLeftPadding,
+              paddingRight: '2rem',
             },
             headerRowStyle: {
               ...tableHeaderStyle,
               ...tableLeftPadding,
+              paddingRight: '2rem',
               marginTop: '1rem',
             },
             cellStyle: {
-              paddingTop: 10,
-              paddingBottom: 10,
+              paddingTop: 13,
+              paddingBottom: 13,
             },
             columns: [
               { header: strong(['Concept name']), key: 'name', size: { grow: 2.3 } },
               { header: strong(['Concept ID']), key: 'id', size: { grow: 0.5 } },
               { header: strong(['Code']), key: 'code', size: { grow: 0.75 } },
               { header: strong(['# Participants']), key: 'count', size: { grow: 0.5 } },
-              { key: 'hierarchy', size: { grow: 0.5 } },
+              { key: 'hierarchy', size: { grow: 0.4 } },
             ],
             rows: _.map((concept) => {
               return {
-                name: div({ style: { display: 'flex' } }, [
+                name: div({ style: { display: 'flex', alignItems: 'start' } }, [
```

Comment:
Should this be `flex-start`? I don't think `start` is a property for flex styling?

Reply:
Ah, good catch! I just tested the difference and for some reason they seem to do the same thing in this instance (not sure why 'start' still works), but I think you're right, 'flex-start' is supposed to be used for flex containers. I'll update it, thank you!

Reply:
I've been playing around with this and I actually think it doesn't make a difference - Originally I had included an align-items 'center', which did change the display and made it so that the checkboxes lined up with the middle of the text, but changed it to 'start' when Lou said he wanted the check boxes to line up with the first line of text. After taking it out though and testing it, I guess the default align-items value has the same effect as align-items start and flex-start because it looks exactly the same. I'll remove it!